sceptre/plan/actions.py

```diff
@@ -1141,3 +1141,10 @@ def _describe_stack_resource_drifts(self) -> dict:
                 "StackName": self.stack.external_name
             }
         )
+
+    @add_stack_hooks
+    def list_config(self, config_reader, command_path):
+        """
+        Returns the Template for the Stack
+        """
+        return config_reader.read(command_path)
```

Comment:
You shouldn't pass the command path here. On StackActions, each method operates on a single stack; The command path might be a reference to a StackGroup. Rather, you need to find a way to invoke the passed config_reader from attributes on `self.stack`. I believe `self.stack.name` will get you most of the way there.